Orange/widgets/data/owrank.py

```diff
@@ -691,7 +691,6 @@ def create_scores_table(self, labels):
 
     @classmethod
     def migrate_settings(cls, settings, version):
-        super().migrate_settings(settings, version)
```

Comment:
Could we use explicit name of the parent class instead of the super() or would that also fail?